src/GitHub.Exports/Helpers/SimpleRepositoryModelExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using GitHub.Models;
+using System;
+using System.Linq;
+using System.IO;
+
+namespace GitHub.Extensions
+{
+    using VisualStudio;
+
+    public static class SimpleRepositoryModelExtensions
+    {
+        public static bool HasCommits(this ISimpleRepositoryModel repository)
+        {
+            var repo = Services.GetRepoFromPath(repository.LocalPath);
+            return repo?.Commits.Count() > 0;
```

Comment:
I generally avoid using `Count` because it has to enumerate the entire enumerable. In this case, we're talking about a LibGit2Sharp queryable which could potentially have costly rev walking implications.

How about?

```csharp
return repo?.Commits != null && (repo.Commits).Any();
```